collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -430,7 +431,7 @@ protected void updateAdapter() {
         mFilteredFormList.clear();
         if (charSequence.length() > 0) {
             for (HashMap<String, String> form : mFormList) {
-                if (form.get(FORMNAME).toLowerCase().contains(charSequence.toString().toLowerCase())) {
+                if (form.get(FORMNAME).toLowerCase(Locale.US).contains(charSequence.toString().toLowerCase(Locale.US))) {
```

Comment:
Don't we want to use the default locale @grzesiek2010 instead of using the US one?